src/PIL/ImageColor.py

```diff
@@ -114,19 +114,20 @@ def getrgb(color):
 
     m = re.match(r"rgba\(\s*(\d+)\s*,\s*(\d+)\s*,\s*(\d+)\s*,\s*(\d+)\s*\)$", color)
     if m:
-        return (int(m.group(1)), int(m.group(2)), int(m.group(3)), int(m.group(4)))
+        return int(m.group(1)), int(m.group(2)), int(m.group(3)), int(m.group(4))
     raise ValueError(f"unknown color specifier: {repr(color)}")
 
 
 def getcolor(color, mode):
     """
     Same as :py:func:`~PIL.ImageColor.getrgb`, but converts the RGB value to a
-    greyscale value if the mode is not color or a palette image. If the string
+    greyscale value if ``mode`` is not color or a palette image. If the string
     cannot be parsed, this function raises a :py:exc:`ValueError` exception.
 
     .. versionadded:: 1.1.4
 
     :param color: A color string
+    :param mode: Convert result to this mode
     :return: ``(graylevel [, alpha]) or (red, green, blue[, alpha])``
```

Comment:
```suggestion
    :return: ``(graylevel[, alpha]) or (red, green, blue[, alpha])``
```
Another "various" change for you, making `(graylevel[, alpha])` match the formatting of `(red, green, blue[, alpha])`